index.php

```diff
@@ -0,0 +1,30 @@
+<?php include('connect.php'); ?><!DOCTYPE html>
```

Comment:
oh, you can compile jade to php?  I've never used it that way...

If you are writing in Jade then you should check your jade files into the repo.  It's more important to have the original files then the compiled files, since you can always generate the compiled files.  If your computer crashed then the jade files would be gone forever.

Optionally you might want to only commit the jade files and add these php files to the .gitignore file